buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -168,6 +168,33 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Create Helix Test Jobs",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "solution": "src\\tests.builds",
```

Comment:
Unfortunately, Tools/msbuild.cmd doesn't actually work unless you've already run the VS Developer Command Prompt script, which we don't here. The other build steps use the run tool to run MSBuild.